test/puppetca_http_api/puppetca_http_impl_test.rb

```diff
@@ -88,7 +111,25 @@ def test_list_puppetserver_6_3
       'puppet.example.com' => {
         'fingerprint' =>
         'F8:DA:15:EA:BD:2F:2D:D3:05:71:73:55:96:74:A4:97:2B:04:06:47:A8:8E:D2:C4:AB:8F:EC:3B:7C:0F:0A:EE',
-          'not_after' => '2019-08-25T19:25:29UTC',
+          'not_after' => '2039-08-25T19:25:29UTC',
```

Comment:
I just noticed that the indenting in this hash is off. The fingerprint should be on the same line as its key. Maybe this is a good time to fix it?

Reply:
Fixed.